docs/api/active_directory_memberships.md

```diff
@@ -0,0 +1,33 @@
+
+# nimbleclient.v1.api.active_directory_memberships
+
+
+## ActiveDirectoryMembership
+```python
+ActiveDirectoryMembership(self,
```

Comment:
Move all the parameters into single line similar to other objects (consistency). Is this from generated modules?
**ActiveDirectoryMembership(self, attrs=None, client=None, collection=None)**

Reply:
This is generated from the pydoc to md tool. Most likely because it takes a long list?